### PR TITLE
Upgrade to 2020 cryptosuites.

### PR DESCRIPTION
**Pull request by msporny**
I'm going to make a major release for bedrock-ledger-node after this is merged... all the dependent libraries have stabilized at this point. The last branch-based dependency, `bedrock-ledger-validator-signature` is only used for testing `bedrock-ledger-node`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # bedrock-ledger ChangeLog
```
Comment (dlongley) [inline CHANGELOG.md:1]:
```suggestion
# bedrock-ledger-node ChangeLog
```

File: package.json [hunk 2]
```diff
@@ -34,10 +34,12 @@
...
     "bedrock-injector": "^1.0.0",
     "bedrock-jsonld-document-loader": "^1.0.0",
-    "bedrock-ledger-context": "^15.0.0",
+    "bedrock-ledger-context": "^18.1.0",
     "bedrock-mongodb": "^8.1.0",
     "bedrock-permission": "^3.0.0",
-    "bedrock-validation": "^4.1.0"
+    "bedrock-validation": "^5.1.0",
+    "ed25519-signature-2020-context": "^1.1.0",
+    "x25519-key-agreement-2020-context": "^1.0.0"
```
Comment (dlongley) [inline package.json:42]: What does this context have to do with bedrock-ledger-node?

```suggestion
```

File: test/mocha/50-records-api.js [hunk 3]
```diff
@@ -18,10 +20,13 @@ let signedConfig;
 describe('Records API', () => {
   before(async function() {
     await helpers.prepareDatabase(mockData);
+    const key =
+      await Ed25519VerificationKey2020.from(mockData.keys.authorized);
```
Comment (dlongley) [inline test/mocha/50-records-api.js:24]: Style here and everywhere else is break on params.

```suggestion
    const key = await Ed25519VerificationKey2020.from(
      mockData.keys.authorized);
```

File: test/mocha/helpers.js [hunk 4]
```diff
@@ -174,14 +171,17 @@ async function insertTestData(mockData) {
...
     documentLoader,
     // FIXME: is this the right purpose?
-    purpose: new AssertionProofPurpose(),
-    suite: new RsaSignature2018({
-      creator,
-      key: new RSAKeyPair({privateKeyPem})
+    purpose: new CapabilityInvocation({
+      capability: doc.id || doc.ledger || doc.record.id,
+      capabilityAction: 'create'
+    }),
```
Comment (dlongley) [inline test/mocha/helpers.js:181]: I expect this to have to change a bit in the near future, just noting.

File: test/package.json [hunk 5]
```diff
@@ -29,24 +29,28 @@
...
-    "bedrock-validation": "^4.1.0",
+    "bedrock-validation": "^5.1.0",
     "cross-env": "^7.0.2",
+    "ed25519-signature-2020-context": "^1.1.0",
     "grunt": "^1.0.1",
-    "jsonld-signatures": "^4.5.1",
-    "nyc": "^15.1.0"
+    "jsonld-signatures": "^9.0.0",
+    "nyc": "^15.1.0",
+    "x25519-key-agreement-2020-context": "^1.0.0"
```
Comment (dlongley) [inline test/package.json:54]: Is this used in the tests?

**Review by dlongley — APPROVED** [review]
Probably some things here that will need to change again real soon, but approving what's been upgraded so far. Thanks!

File: test/package.json (hunk 5 above)
Comment (msporny) [inline test/package.json:54]:
```suggestion
    "nyc": "^15.1.0"
```

Comment (msporny) [reply test/package.json:54]: No, it isn't... removing.

**Pull request merged**